core/src/main/scala/org/apache/spark/sql/SnappyContext.scala

```diff
@@ -1112,7 +1112,8 @@ object SnappyContext extends Logging {
                   val commandSet = ToolsCallbackInit.toolsCallback.getGlobalCmndsSet
                   commandSet.forEach(new Consumer[Entry[String, String]] {
                     override def accept(t: Entry[String, String]): Unit = {
-                      if (!t.getKey.startsWith(ContextJarUtils.functionKeyPrefix)) {
+                      if (!t.getKey.startsWith(ContextJarUtils.functionKeyPrefix)
+                        && !t.getKey.startsWith("__")) {
```

Comment:
Please put the pattern used as a single place so that anyone going to use the meta region for new purpose is aware of what patterns to use. Please define a set of prefix Constants which one can use for their purpose and use that instead of herd coding. Like It is being used for jars and packages now, So something like jar_name or package_name can be the key. You can use it with prefix clusterid__key .. Going forward others can use it too provided they have a guideline to do it.

Reply:
Also, add this pattern in the `if` case at the end of `RecoveryService.getAllDDLs()`. Check all such references of `ContextJarUtils.functionKeyPrefix` and make sure you include your new pattern also, wherever required.

Reply:
Added suggested change